libs/blocks/text/text.css

```diff
@@ -181,15 +185,25 @@
   .inset.text-block .foreground > div {
     padding-left: var(--spacing-l);
   }
-  
+
   .link-farm.text-block .foreground {
     grid-template-columns: repeat(2, minmax(0, 1fr));
   }
+
+  .link-farm.text-block [id^="no-heading"] {
+    height: var(--type-heading-m-size);
+    margin: 0 0 var(--spacing-l);
+  }
 }
 
 /* Desktop up */
 @media screen and (min-width: 1200px) {
   .link-farm.text-block .foreground {
     grid-template-columns: repeat(4, minmax(0, 1fr));
   }
+
+  .link-farm.text-block [id^="no-heading"] {
+    height: var(--type-heading-m-size);
+    margin: 0 0 var(--spacing-l);
+  }
```

Comment:
I don't think you need this block since you add the same selectors and properties in the 1min-width: 600` media query.

Reply:
Resolved. Thanks for pointing out!